services/core/shared/core/compat/common/constants.js

```diff
@@ -23,11 +24,19 @@ let coreVersion = '1.0.0-alpha.0';
 let readyStatus;
 
 const getNetworkConstants = async () => {
-	// const expireMiliseconds = Number(config.ttl.stable) * 1000;
-	// const result = await coreApiCached.getNetworkConstants(null, { expireMiliseconds });
-	const result = await http.get('/node/constants'); // Necessary to remove cyclic dependency
-	if (!isProperObject(result)) return {};
-	return result;
+	try {
+		let result = await http.get('/node/constants'); // Necessary to remove cyclic dependency
+		if (Object.getOwnPropertyNames(result).length === 0) {
+			const apiClient = await ws.getClient();
```

Comment:
Is the node URL taken from the config?

Reply:
Yes, node URL is taken from config file and file `services/core/shared/core/compat/common/wsRequest.js` is responsible for it.
```
const { createWSClient } = require('@liskhq/lisk-api-client');
const config = require('../../../../config');

const liskAddress = config.endpoints.liskWs;
let clientCache;

const getClient = async () => {
    try {
        clientCache = await createWSClient(`${liskAddress}/ws`);
        return clientCache;
    } catch (err) {
        return {
            data: { error: 'Action not supported' },
            status: 'METHOD_NOT_ALLOWED',
        };
    }
};```